src/stories/components/404/not-found-component-404.tsx

```diff
@@ -0,0 +1,308 @@
+  paddingLeft: '32px',
+  maxWidth: '343px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    maxWidth: '100%',
+  },
+});
+
+const TextUps = styled(Typography)<{ isLight?: boolean }>(({ isLight }) => ({
+  fontFamily: 'FT Base, sans-serif',
+  fontStyle: 'normal',
+  fontWeight: 700,
+  fontSize: '36px',
+  lineHeight: '43px',
+  letterSpacing: '0.4px',
+  color: isLight ? '#787A9B' : '#D2D4EF',
+  textAlign: 'center',
+  marginBottom: '24px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    fontWeight: 700,
+    fontSize: '48px',
+    lineHeight: '58px',
+    marginBottom: '32px',
+  }
+}));
+
+const TextDescription = styled(Typography)<{ isLight?: boolean }>(({ isLight }) => ({
+  fontFamily: 'FT Base,san-serif',
+  fontStyle: 'normal',
+  fontWeight: 500,
+  fontSize: '20px',
+  lineHeight: '24px',
+  textAlign: 'center',
+  letterSpacing: '0.4px',
+  color: isLight ? '#ADAFD4' : '#ADAFD4',
+  marginBottom: '64px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    fontSize: '24px',
+    lineHeight: '29px',
+    marginBottom: '40px',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    fontSize: '24px',
+    lineHeight: '29px',
+    marginBottom: '40px',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    fontSize: '24px',
+    lineHeight: '29px',
+    marginBottom: '40px',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    fontSize: '24px',
+    lineHeight: '29px',
+    marginBottom: '40px',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    fontSize: '24px',
+    lineHeight: '29px',
+    marginBottom: '40px',
+  }
+}));
+
+const ContainerButton = styled.div({
+  marginBottom: '83px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
```

Comment:
Same recommendation about media. Please, note that the previous media is empty

Reply:
Fixed